src/PickerInput/RangePicker.tsx

```diff
@@ -634,6 +634,8 @@ function RangePicker<DateType extends object = any>(
 
   const onSelectorBlur: SelectorProps['onBlur'] = (event, index) => {
     triggerOpen(false);
+    const nextIndex = nextActiveIndex(calendarValue);
```

Comment:
Wrap of `needConfirm` check? When `needConfirm` blur is not a valid submit operation.

Reply:
Yeah, you're absolutely right. Haven't thought about it. Thank you

Wrapped with `needConfirm` check